packages/roc-package-web-app-react/package.json

```diff
@@ -31,27 +31,22 @@
   "dependencies": {
     "debug": "~2.2.0",
     "error": "~7.0.2",
-    "history": "~1.17.0",
-    "nunjucks": "~2.1.0",
-    "pretty-error": "~1.2.0",
-    "react-a11y": "~0.2.6",
-    "react-fetcher": "~0.2.0",
+    "history": "^2.1.0",
+    "nunjucks": "~2.4.2",
+    "pretty-error": "~2.0.0",
     "react-helmet": "~2.3.1",
-    "react-proxy": "1.1.8",
-    "react-redux": "~4.0.0",
-    "react-router": "~1.0.0",
+    "react-redux": "~4.4.5",
+    "react-router": "~2.3.0",
+    "react-router-redux": "4.0.2",
```

Comment:
That should be a `~`, I have changed my default setting for npm and forgot to update this.